tests/test_keyboard_shortcuts.py

```diff
@@ -7,6 +7,53 @@
 from pages.treeherder import TreeherderPage
 
 
+def test_close_open_panels(base_url, selenium):
+    """Open Treeherder, verify shortcut: 'Esc' closes filter and job panel.
+    Open Treeherder page, open Filters panel, select random job, close all
+    panels using 'esc' button, verify if all panels are closed.
+    """
+    page = TreeherderPage(selenium, base_url).open()
+
+    page.click_on_filters_panel()
+    page.select_random_job()
+
+    assert page.filter_panel_is_open
+    assert page.info_panel_is_open
```

Comment:
I would use region objects for the filter panel and info panel, to prevent the page object becoming overloaded. Also, there's already a job details region, which is within the info panel.